cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -10,4 +12,33 @@ def digital_specific_transformations(df):
     df = df.withColumn("visit_id", F.col("participant_completion_window_id"))
     df = df.withColumn("visit_datetime", F.col("digital_enrolment_invite_datetime"))
     df = transform_survey_responses_generic(df)
+    df = assign_raw_copies(
+        df,
+        [
+            "work_in_additional_paid_employment",
+            "self_isolating",
+            "illness_lasting_over_12_months",
+            "ever_smoked_regularly",
+            "currently_smokes_or_vapes",
+            "hours_a_day_with_someone_else_at_home",
+            "face_covering_work_or_education",
+        ],
+    )
+    mapping_dict = {
+        "Prefer not to say": None,
+        "Don&#39;t Know": None,
+        "Don&amp;#39;t Know": None,
+    }
+    df = apply_value_map_multiple_columns(
+        df,
+        {
+            "work_in_additional_paid_employment": mapping_dict,
```

Comment:
Would be best to refactor the duplicate lists here into a single list.

This is also missing variables from the document linked in the spec. I'm not sure if I'm filtering to catch them all either, so please get in touch with those who wrote the spec to review which are included